sdk/purview/Azure.Analytics.Purview.Workflows/src/Generated/WorkflowsClientBuilderExtensions.cs

```diff
@@ -12,7 +12,7 @@
 namespace Microsoft.Extensions.Azure
 {
     /// <summary> Extension methods to add <see cref="PurviewWorkflowServiceClient"/> to client builder. </summary>
-    public static partial class PurviewWorkflowServiceClientBuilderExtensions
+    public static partial class WorkflowsClientBuilderExtensions
```

Comment:
changed to `PurviewWorkflowsClientBuilderExtensions` to align with others
This should be OK since this is beta.